### PR TITLE
Add Abeeway devices

### PR DESCRIPTION
**Pull request by ymgupta**
#### Summary
Added Abeeway devices as per request from the vendor.

#### Changes
- Added 
    - abeeway-industrial-tracker
    - abeeway-compact-tracker
    - abeeway-micro-tracker
    - abeeway-smart-badge
    - abeeway-geolocation-module
- Added `Abeeway` to vendors list.

### CODE REVIEW

**Review by johanstokking — COMMENTED** [review]
The `vendorProfileID` is probably wrong; please set to `0`

Are there no codecs available?

**Comment by johanstokking** [comment]
Also please rebase to resolve conflict

**Comment by ymgupta** [comment]
@johanstokking, Made the changes as suggested.

> Are there no codecs available?

The vendor mentioned that the Abeeway driver is based on Nodejs. Their nodeJS scripts are not available for public use, and the driver is available to be used as a docker image or as an online service.

File: vendor/abeeway/abeeway-compact-tracker.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+name: Abeeway Compact Tracker
+description: Higly versatile multi-mode tracke r with a compact and robust form factor, designed to resist harsh surroundings enabling you to perform seamless asset tracking and management in any industrial environment – whether it’s for heavy-duty assets or small equipment, taking advantage of its long battery autonomy.
```
Comment (benolayinka) [inline vendor/abeeway/abeeway-compact-tracker.yaml:2]:
```suggestion
description: Higly versatile multi-mode tracker with a compact and robust form factor, designed to resist harsh surroundings enabling you to perform seamless asset tracking and management in any industrial environment – whether it’s for heavy-duty assets or small equipment, taking advantage of its long battery autonomy.
```

File: vendor/abeeway/abeeway-geolocation-module.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
+name: Abeeway Geolocation Module
+description: The Abeeway   Geolocation   Module   is   a   multi-technology   fused   location  LPWAN  module  designed  as  the  ideal  platform  to  develop a  wide  range  of  IoT  tracking  solutions,  minimizing  the  Total  Cost  of  Ownership  (TCO)  of  IoT  geolocation  use  cases.  It  allows  your  engineering  to  focus  on  the  use  case,  to  optimize  time-to-market  and  cost  by  using  less  complex  PCBs  (all  high-density  devices  being  encapsulated), to reduce certification risk and budget, as well as testing time and cost for mass production.
```
Comment (benolayinka) [inline vendor/abeeway/abeeway-geolocation-module.yaml:2]: Lots of extra spaces

**Review by benolayinka — APPROVED** [review]

**Pull request merged**